packages/landing/src/map.ts

```diff
@@ -0,0 +1,107 @@
+import Map from 'ol/Map';
+import { WindowUrl, MapOptions, MapOptionType, MapLocation } from './url';
+import View from 'ol/View';
+import TileLayer from 'ol/layer/Tile';
+import XYZ from 'ol/source/XYZ';
+import * as proj from 'ol/proj.js';
+import WMTS from 'ol/source/WMTS.js';
+import { Epsg } from '@basemaps/geo';
+import { NztmOl } from './nztm2000';
+import Layer from 'ol/layer/Layer';
+
+/** Projection to use for the URL bar */
+const UrlProjection = 'EPSG:4326';
+
+/** Default center point if none provided */
+const DefaultCenter: Record<number, MapLocation> = {
+    [Epsg.Google.code]: { lat: -41.277848, lon: 174.7763921, zoom: 8 },
+    [Epsg.Nztm2000.code]: { lat: -41.277848, lon: 174.6763921, zoom: 2 },
+};
+
+const attributions = '<a href="http://data.linz.govt.nz">LINZ CC BY 4.0 © Imagery Basemap contributors</a>';
```

Comment:
Maybe for now this can be a link https://www.linz.govt.nz/data/licensing-and-using-data/attributing-elevation-or-aerial-imagery-data?

Reply:
There is a snippet somewhere that vicki has made, just hadnt got around to adding it in yet. I believe it links to somewhere like that.

I took this snippet from a codepen I think you gave me

Reply:
> I took this snippet from a codepen I think you gave me

yeah that codepend snippet was a copy and paste for general LDS with some random basemaps text!

Should so something like

```html
<a href=" https://www.linz.govt.nz/data/licensing-and-using-data/attributing-elevation-or-aerial-imagery-data">Sourced from LINZ. CC BY 4.0</a>'
```

Reply:
Cool, updated.